src/eval-frame/actions/__tests__/fetch-eval-actions.test.js

```diff
@@ -0,0 +1,5 @@
+describe('stub', () => {
```

Comment:
want to add some tests around "fetch-cell-eval-actions.js"? i put this stub in place, but for the stuff id had written so far i wasn't sure how to test it (fetch and promise stuff). if you know how to write tests around that, it would be awesome (it looks like some of these functions are pure enough that it might not be too bad?)

if not, let's delete this file.

Reply:
Going to delete this file for now, I think. It's not immediately obvious how to test this one action as-is.

Reply:
That said, I do think it's worth getting the actions better tested (see #1054) I just don't know if this is the PR to begin tackling that.